inst/tests/other.Rraw

```diff
@@ -699,6 +699,14 @@ if (loaded[["nanotime"]]) {
   DT = data.table(time=nanotime(c(1,NA,3)))
   test(27, na.omit(DT), DT[c(1,3)])
 
+  # was 2239 in tests.Rraw, rbind with vectors with class attributes #5309
```

Comment:
is there more context here? I see this as 2239 now (L18210):

```r
test(2239.1, shift(matrix(1:10, ncol = 1)), error="consider wrapping")
```

Reply:
The original test case number for this PR was 2239, but yes it sounded confusing